data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -73,3 +73,29 @@ spec:
         - name: cnibin
           hostPath:
             path: {{ .CNIBinDir }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: passt-binding-cni
+  namespace: kube-system
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: passt-binding-cni
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+readOnlyRootFilesystem: false
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+users:
+- system:serviceaccount:kube-system:passt-binding-cni
+volumes:
+- "*"
```

Comment:
missing new line

do we need please to wrap with SCC enabled ? as we have on 001-rbac file